public/app/features/dashboard-scene/scene/Scopes/ScopesFiltersBasicSelectorScene.tsx

```diff
@@ -0,0 +1,133 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import {
+  SceneComponentProps,
+  SceneObjectUrlSyncConfig,
+  SceneObjectUrlValues,
+  SceneObjectWithUrlSync,
+} from '@grafana/scenes';
+import { Icon, Input, Toggletip } from '@grafana/ui';
+import { useStyles2 } from '@grafana/ui/';
+
+import { ScopesFiltersBaseSelectorScene } from './ScopesFiltersBaseSelectorScene';
+import { ScopesTreeLevel } from './ScopesTreeLevel';
+import { ScopesFiltersOpenAdvanced } from './events';
+
+export class ScopesFiltersBasicSelectorScene extends ScopesFiltersBaseSelectorScene implements SceneObjectWithUrlSync {
+  static Component = ScopesFiltersBasicSelectorSceneRenderer;
+
+  protected _urlSync = new SceneObjectUrlSyncConfig(this, { keys: ['scopes'] });
+
+  constructor() {
+    super();
+
+    this.toggleNodeSelect = this.toggleNodeSelect.bind(this);
+    this.openAdvancedSelector = this.openAdvancedSelector.bind(this);
+  }
+
+  getUrlState() {
+    return { scopes: this.state.scopes.map((scope) => scope.metadata.name) };
+  }
+
+  updateFromUrl(values: SceneObjectUrlValues) {
+    let scopesNames = values.scopes ?? [];
+    scopesNames = Array.isArray(scopesNames) ? scopesNames : [scopesNames];
+
+    Promise.all(scopesNames.map((scopeName) => this.fetchScope(scopeName))).then((scopes) => this.setState({ scopes }));
```

Comment:
This may lead to funky problems, i.e for slower connections fetching scope may possibly take a while, while in the meantime the user can already start interacting with the scope selector and apply scopes.